src/components/screen/Examples/Oauth.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div id="app">
+    <h1>Press button to request email</h1>
+    <h2>Result: {{ email }}</h2>
+    <button @click="onClickRequestButton">
+      Request email
+    </button>
+  </div>
+</template>
+
+<script>
+import EndpassOauthPlugin from '@endpass/connect/oauth';
+
+const connect = new EndpassOauthPlugin({
+  authUrl: 'https://auth.endpass.com',
```

Comment:
Don't need authUrl, oauthServer. Can we get oauth id and request base url from ENV?